src/components/Avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import React from 'react';
+import Avatar from '../Avatar/Avatar';
+
+export default {
+    title: 'UI/Avatar',
+    component: Avatar,
+} as ComponentMeta<typeof Avatar>;
+
+const Template: ComponentStory<typeof Avatar> = (args) => <Avatar {...args} />;
+
+export const AvatarImageDefault = Template.bind({});
+AvatarImageDefault.args = {
+    theme: 'image',
+};
+
+export const AvatarImageCustom = Template.bind({});
+AvatarImageCustom.args = {
+    theme: 'image',
+    image: 'https://scalebranding.com/wp-content/uploads/2021/06/penguin.jpg',
```

Comment:
maybe we should point this to an image we host?

Reply:
Adding an image inside the component `Avatar/images` and using that one

Reply:
Amended as suggested.